include/oneapi/dnnl/dnnl_ocl.h

```diff
@@ -75,7 +75,6 @@ dnnl_status_t DNNL_API dnnl_ocl_interop_memory_create(dnnl_memory_t *memory,
         const_dnnl_memory_desc_t memory_desc, dnnl_engine_t engine,
         dnnl_ocl_interop_memory_kind_t memory_kind, void *handle);
 
-#ifdef DNNL_EXPERIMENTAL_SPARSE
```

Comment:
We don't have `get_mem_object` and `set_mem_object` functions that would take indices. I don't think it's critical, but just need to have that in mind as we may have to add it later on.